src/stampy_nlp/utilities/coda_utils.py

```diff
@@ -71,6 +77,8 @@ def get_df_data(is_similar = lambda n1, n2: n1 == n2, delete_all: bool = False):
                     'url': url,
                 })
 
+    df_similar = pd.DataFrame(similar_alternates)
+    df_similar.to_csv("similar_alternates.tsv", sep='\t', index=False)
```

Comment:
Adding a TSV file of skipped similar alternative phrasings for my own record. Might publish it eventually if it gets too large. Someone has been getting a little overly zealous with the list:

I’d like to get deeper into the AI alignment literature. Where should I start?
I’d like to delve deeper into the AI alignment literature. Where should I look?
I’d like to delve deeper into the AI alignment literature. Where should I start?
I’d like to take a deep dive into the AI alignment literature. Where should I look?
I’d like to take a deep dive into the AI alignment literature. Where should I start?
What’s a good starting point for exploring the AI alignment literature?
What’s a good starting point for exploring the alignment literature?